src/dbcp/metadata/data_mart.py

```diff
@@ -529,64 +527,46 @@
     schema=schema,
 )
 
-iso_counties_new_clean_n_projects_changelog = Table(
-    "iso_counties_new_clean_n_projects_changelog",
+counties_active_projects_capacity_mw_change_log = Table(
```

Comment:
Could these 4 tables be condensed to 2? I see two tables with PK (`county_id_fips`, `report_date`) and two with PK (`iso_region`, `report_date`). Or was this format a specific request?

Reply:
I proposed combining the tables but they requested four separate tables.